Automattic/simplenote-macos
# Issue/notarization

### Fix
This PR enables `hardened-runtime` for the beta version of the App, so that it can be submitted for notarization to Apple.
For the same reason, this PR also resigns  the Sparkle binaries with the `hardened-runtime` flag.

**Note:** It looks like `fastlane` has some issues with signing Sparkle with `developer-id`, so this PR is not updating the `Fastfile`. Since we want to release a new beta of Simplenote during this Sprint, I've opened an issue [here](https://github.com/Automattic/simplenote-macos/issues/497) so that we'll look into it later. In the meantime, we have to switch back to manual builds with XCode. cc/ @mokagio. 

### Test
1. In XCode, build and archive the app.
2. Go to Organizer, select the build, click on Distribute->Developer ID->Upload.
3. Select the `Simplenote MacOS beta` profile and upload for notarization.
4. Verify that the app is successfully notarized.
5. Version `1.10.0.1` on AppCenter has been built out of this branch. Download it and verify that it can be opened on Catalina. 

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
